core/src/main/java/com/predic8/membrane/core/interceptor/lang/AbstractSetterInterceptor.java

```
@@ -63,24 +62,20 @@ private Outcome handleInternal(Exchange exchange, Flow flow) {
         if (!shouldSetValue(exchange, flow))
             return CONTINUE;
 
-        String msg;
         try {
             setValue(exchange, flow, exchangeExpression.evaluate(exchange, flow, Object.class));
-            return CONTINUE;
-        catch (Exception e) {
-            msg = e.getMessage();
+        } catch (Exception e) {
```

The `Exception`'s message is swallowed, but might indicate errors in the expression (e.g. syntax errors).

---

will be fixed in the call-example branch